highs/mip/HighsSearch.cpp

```diff
@@ -530,18 +531,13 @@ HighsInt HighsSearch::selectBranchingCandidate(int64_t maxSbIters,
       }
     };
 
-    if (!downscorereliable[candidate] &&
-        (upscorereliable[candidate] ||
-         std::make_pair(downscore[candidate],
-                        pseudocost.getAvgInferencesDown(col)) >=
-             std::make_pair(upscore[candidate],
-                            pseudocost.getAvgInferencesUp(col)))) {
-      // evaluate down branch
-      // if (!mipsolver.submip)
-      //   printf("down eval col=%d fracval=%g\n", col, fracval);
+    auto strongBranch = [&](bool upbranch) {
```

Comment:
Consider adding the return type of this lambda with `->` for clarity.

[C++ Reference on Lambda expressions](https://en.cppreference.com/w/cpp/language/lambda.html#:~:text=cannot%20be%20used).-,trailing,-%2D)

Reply:
Added